autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -137,10 +139,12 @@ def run(self):
             allow_methods=["*"],  # Allows all methods
             allow_headers=["*"],  # Allows all headers
         )
+        log_config = generate_uvicorn_config()
         uvicorn.run(
             server_app,
             host=backend.util.settings.Config().agent_api_host,
             port=backend.util.settings.Config().agent_api_port,
+            log_config=log_config,
```

Comment:
nit: why the need of local variable, we can assign it directly to the log_config param